From cc05b137bbcb93926f93a96521000a4cb14fa153 Mon Sep 17 00:00:00 2001
From: Sudarshan Rajagopalan <sudaraja@codeaurora.org>
Date: Wed, 28 Oct 2020 14:31:09 -0700
Subject: [PATCH 226/830] ANDROID: mm/memblock: export memblock_end_of_DRAM
X-NVConfidentiality: public

Export memblock_end_of_DRAM() so that module drivers can know
the physical end address of memory blocks that system booted with.

Bug: 171499373
Change-Id: Ib0c90b736f16d1abbf47d4f3288443c35f9e17b4
Signed-off-by: Sudarshan Rajagopalan <sudaraja@codeaurora.org>
---
 mm/memblock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/memblock.c b/mm/memblock.c
index f72d53957033..acda689e473b 100644
--- a/mm/memblock.c
+++ b/mm/memblock.c
@@ -1628,6 +1628,7 @@ phys_addr_t __init_memblock memblock_end_of_DRAM(void)
 
 	return (memblock.memory.regions[idx].base + memblock.memory.regions[idx].size);
 }
+EXPORT_SYMBOL_GPL(memblock_end_of_DRAM);
 
 static phys_addr_t __init_memblock __find_max_addr(phys_addr_t limit)
 {
-- 
2.17.1

